utils/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/AvroDefaultExpressionExtractorTest.scala

```diff
@@ -0,0 +1,106 @@
+package pl.touk.nussknacker.engine.avro
+
+import cats.data.Validated.Invalid
+import cats.data.ValidatedNel
+import org.scalatest.{Assertion, FunSuite, Matchers}
+import pl.touk.nussknacker.engine.avro.AvroDefaultExpressionExtractor.AvroDefaultToSpELExpressionError
+import pl.touk.nussknacker.engine.graph.expression.Expression
+
+class AvroDefaultExpressionExtractorTest extends FunSuite with Matchers {
+  import pl.touk.nussknacker.engine.spel.Implicits.asSpelExpression
+
+  test("string default") {
+    val stringField = schema.getFields.get(0)
+    val expression = new AvroDefaultExpressionExtractor(stringField, handleNotSupported = false).toExpression
+
+    verify(expression) { _ shouldBe Some(asSpelExpression("'stringDefault'")) }
```

Comment:
:+1:  I've added assertion to check wheater the evaluated spel expression matches excpected default value.

Reply:
The problem is that you don't know what is expected value - avro will know. Let say that someone add support for local time. He/she will generate expression like `LocalTime.ofSecondOfDay` then he/she will add verfication if value is equals to result of `LocalTime.ofSecondOfDay` - test will passes but the code would be wrong because he/she should use `LocalTime.ofNanoOfDay`. If you will check what returns code with `LogicalTypesGenericRecordBuilder` you will see how avro handles defaults on their own - and it is expected value

Reply:
Ur right. Now I'm using LogicalTypesGenericRecordBuilder.

Reply:
Ur right. Now I'm using LogicalTypesGenericRecordBuilder.